code/modules/projectiles/guns/projectile/pistol.dm

```diff
@@ -227,8 +227,8 @@
 		/obj/item/ammo_casing/shotgun           = "12 gauge",
 		/obj/item/ammo_casing/shotgun/pellet    = "12 gauge",
 		/obj/item/ammo_casing/shotgun/pellet    = "12 gauge",

```

Comment:
No idea what was the original purpose when some bay code added it, I guess it was to make them more common than others?